### ISSUE TITLE
switch the order of latitude and longitude

### ISSUE THREAD

**Issue by panavrin** [echo-lab/SensDat#22]
Now it shows longitude first
Let's switch the order.

**Comment by Andy1uu** [comment]
So the table that is in the main page would just need the order of these two columns switched?

**Comment by maneshd** [comment]
Yep! The columns already have some sense of ordering as specified [here](https://github.com/echo-lab/SensDat/blob/a8654106bb79ee343d4482a573dda0b355718086/src/data-table.js#L24), so you'd just need to change that. 

I actually think it would be super nice to squish Latitude/Longitude into the same column, too. e.g., have a column called `Latitude/Longitude` with rows like `(37.2299104, -80.4257024)`.  It'd save some space in the table. This one is harder to do w/ the table-rendering library, so don't worry if you don't get to it - you can just do the first part.

**Comment by Andy1uu** [comment]
I have switched the order by changing the order in the array and calling the sort method when when the table is finished loading. Please look over the PR when you get the chance. Thank you.
